tests_e2e/pipeline/templates/execute-tests.yml

```diff
@@ -26,20 +26,23 @@ jobs:
 
       - bash: $(Build.SourcesDirectory)/tests_e2e/pipeline/scripts/execute_tests.sh
         displayName: "Execute tests"
+        continueOnError: true
```

Comment:
Test failures should not stop the pipeline, so we allow this task to continue on error